pmd-core/src/main/java/net/sourceforge/pmd/RuleSetFactory.java

```diff
@@ -848,15 +841,15 @@ private static String valueFrom(Node parentNode) {
     private static void parsePropertyNodeBR(Rule rule, Node propertyNode) {
 
         Element propertyElement = (Element) propertyNode;
-        String typeId = propertyElement.getAttribute(PropertyDescriptorFields.TYPE);
-        String strValue = propertyElement.getAttribute(PropertyDescriptorFields.VALUE);
+        String typeId = propertyElement.getAttribute(PropertyDescriptorField.TYPE.attributeName);
```

Comment:
There is a bunch of commented out PropertyDescriptor related code 10+ lines above this comment.  Should it be cleaned up?